ddpui/tests/services/test_chart_service.py

```
@@ -0,0 +1,377 @@
+            d.delete()
+
+
+# ================================================================================
+# Test get_org_warehouse
+# ================================================================================
+
+
+class TestGetOrgWarehouse:
+    """Tests for WarehouseService.get_org_warehouse()"""
+
+    def test_get_org_warehouse_success(self, org, seed_db):
```

Ruff (0.14.8)

Redefinition of unused `seed_db` from line 35

(F811)